data/json/monsters/feral_humans.json

```
@@ -1019,5 +1019,119 @@
       "PATH_AVOID_DANGER_1",
       "DROPS_AMMO"
     ]
+  },
+  {
+    "id": "mon_feral_big_pappy",
+    "type": "MONSTER",
+    "name": { "str": "Big Pappy", "str_pl": "Big Pappys" },
```

```suggestion
    "name": { "str": "Big Pappy" },
```
`s` is added automatically.

---

Yep - it failed it again haha

---

Possibly because it ends in a `y`?

---

Let me know if you want me to re-add it.

---

Done!